.node-version

```diff
@@ -0,0 +1 @@
+14.15.1
```

Comment:
What is this file for :eyes:

Reply:
lol its removed now. its for `node-env` -.-